include/seqan/journaled_set/score_biaffine.h

```diff
@@ -90,13 +90,6 @@ class Score<TScoreValue, BiAffine>
                                                                         _gapOpenHorizontal(gapOpenHorizontal),
                                                                         _gapExtendVertical(gapExtendVertical),
                                                                         _gapOpenVertical(gapOpenVertical) {}
-
-    Score(Score const & other) : _match(other._match),
-                                 _mismatch(other._mismatch),
-                                 _gapExtendHorizontal(other._gapExtendHorizontal),
-                                 _gapOpenHorizontal(other._gapOpenHorizontal),
-                                 _gapExtendVertical(other._gapExtendVertical),
-                                 _gapOpenVertical(other._gapOpenVertical) {}
 };
```

Comment:
I think there definitely should be a copy constructor, but it could also be defaulted. TBH, if you change this you could as well make it an aggregate type and remove all the constructors. But this require changes in other places...